assets/sass/okta-sign-in.scss

```diff
@@ -162,3 +161,5 @@
     background: transparent;
   }
 }
+
```

Comment:
We are probably processing this with SASS so it may not matter. But if we switched to a lighter CSS processing system, like postCSS. Technically, all @import statements should precede all other rules: https://developer.mozilla.org/en-US/docs/Web/CSS/@import

In this case, the login rule should be @imported from a new file.

Reply:
sounds an bigger task to refactor the scss file ?

Reply:
@aarongranick-okta Can you create a Jira if you feel this is something we should do in future? Thank you